pkg/transformers/json/transformer.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package json
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/mainflux/mainflux/pkg/errors"
+	"github.com/mainflux/mainflux/pkg/messaging"
+	"github.com/mainflux/mainflux/pkg/transformers"
+)
+
+const sep = "/"
+
+var (
+
```

Comment:
Rm this blank line

Reply:
Do we want to add this global `var` to a var group below? Or you separated them deliberately?